Base/usr/share/man/man4/passwd.md

```diff
@@ -0,0 +1,29 @@
+## Name
+
+passwd - Users on the system
+
+## Description
+
+`/etc/passwd` is a text file where each line represents a user on the system, with
+colon-separated fields such as the home directory or the default shell:
+- The first field is the username.
+- The second field is the encrypted password. It is always a `!`, to indicate that the real password is in `/etc/shadow`.
+- The third field is the user ID.
+- The fourth field is the main group ID.
+- The fifth field is the GECOS username, a longer string to indicate general information about the user.
```

Comment:
.. username; a longer string (use semicolon)

Reply:
what about a colon? the two people i've asked say it sounds better with a colon